src/eda.py

```diff
@@ -44,19 +44,19 @@ def create_and_save_chart(df, col, save_path):
 
     Examples
     --------
-    >>> create_chart('C_WTHR', 'Weather condition')
+    >>> create_chart(train_df, 'C_WTHR', results)
     """
 
-    if col == "P_AGE":
+    if col == "P_SEX":
```

Comment:
Will P_AGE work properly if you replace the if statement with P_SEX?

Reply:
not applicable to new code